README.md

```diff
@@ -14,13 +18,37 @@ connections and responses from end endpoint.
 
 (TODO: some words to go along with these diagrams)
 
-#### Development setup
+### Development setup
+
+#### [Node.js](http://nodejs.org)
+
+You may want to use [nvm](https://github.com/creationix/nvm) as your maanger of Node versions on your
+local machine.  Think RVM for Node.js
```

Comment:
Looks like 2 spaces between the sentences, let's just have one. Also needs a period at the end of the second sentence (sorry, I was an English Lit major, I can't help myself).